quickwit-cli/src/index.rs

```diff
@@ -480,6 +522,26 @@ impl IndexCliCommand {
     }
 }
 
+pub async fn list_index_cli(args: ListIndexesArgs) -> anyhow::Result<()> {
+    debug!(args = ?args, "describe");
+    let metastore_uri_resolver = quickwit_metastore_uri_resolver();
+    let quickwit_config = load_quickwit_config(&args.config_uri, args.data_dir).await?;
+    let metastore_uri = if let Some(uri) = args.metastore_uri {
+        uri.to_string()
+    } else {
+        quickwit_config.metastore_uri()
+    };
+    let metastore = metastore_uri_resolver.resolve(&metastore_uri).await?;
+    let res = metastore.list_indexes_metadatas().await?;
+
+    println!();
```

Comment:
Can we display the indexes as a table? There are examples in the source and split commands.

| Indexes |
| Index ID | Index URI |

Reply:
Indexes                                     
+-----------+-------------------------------------------------------------------+
| Index ID  |                             Index URI                             |
+-----------+-------------------------------------------------------------------+
| hdfs-logs | file:///home/dai/Desktop/quickwit-v0.2.0/qwdata/indexes/hdfs-logs |
+-----------+-------------------------------------------------------------------+
| wikipedia | file:///home/dai/Desktop/quickwit-v0.2.0/qwdata/indexes/wikipedia |
+-----------+-------------------------------------------------------------------+

Thanks i added the update